truss-chains/truss_chains/utils.py

```diff
@@ -128,6 +130,25 @@ def get_free_port() -> int:
         return port
 
 
+def override_chainlet_to_service_metadata(
```

Comment:
`override` implies that we're modifying the passed in data structure (which we are). I wouldn't expect us to return the data structure we're modifying...Is it necessary to return the mapping?